modules/canbus/vehicle/ch/protocol/BUILD

```diff
@@ -0,0 +1,214 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "canbus_ch_protocol",
```

Comment:
Please separate them in the BUILD file, one library for each.

Reply:
BUILD file has been modified as you required, please help to review again, thank you

Reply:
@wfgsy101 Did you commit the latest one? It still looks like they are put into one binary

Reply:
@Capri2014 the latest BUILD as below, is there still anything wrong,

<img width="624" alt="cc77474535032bf50aa8788d1466b82a" src="https://user-images.githubusercontent.com/40455045/55299710-bb764b00-5466-11e9-973b-4d57ec1a0d6f.png">

Reply:
`canbus_ch_protocol` should be depending on the binaries then.

Reply:
Take this one as a reference: https://github.com/ApolloAuto/apollo/blob/812d3e396d48dbc3b50b7eab9af26f8cffabb546/modules/canbus/vehicle/transit/protocol/BUILD#L8

Reply:
Sorry I mean *libraries.

Reply:
@Capri2014  update the BUILD，please review,  thanks